src/main/java/org/jabref/gui/collab/stringrename/StringRenameDetailsView.java

```diff
@@ -0,0 +1,18 @@
+package org.jabref.gui.collab.stringrename;
+
+import javafx.scene.control.Label;
+
+import org.jabref.gui.collab.ExternalChangeDetailsView;
+
+public final class StringRenameDetailsView extends ExternalChangeDetailsView {
+
+    public StringRenameDetailsView(StringRename stringRename) {
+        Label label = new Label(stringRename.getNewString().getName() + " : " + stringRename.getOldString().getContent());
```

Comment:
Maybe some comment about what is to be displayed here... huh?

Reply:
This view shows the state of the string after it is renamed, so if we have a string `oldString : 1` and we rename it to `newString` then this should display `newString : 1`. I believe the use of `stringRename.getOldString().getContent()` is what causes the confusion. So I think using `getNewString()` might be a better fix than a comment.

```java
Label label = new Label(stringRename.getNewString().getName() + " : " + stringRename.getNewString().getContent());
```

Reply:
Since this PR has already been merged, this could be addressed in another PR while working on the external changes resolver dialog.

Reply:
Yes, you can change the name as a side quest to the test PR.